.github/workflows/ci.yml

```diff
@@ -154,6 +154,9 @@ jobs:
             - { target: x86_64-linux-android,             os: ubuntu-latest,  cpp: 1,           std: 1, run: 1 }
             - { target: x86_64-pc-windows-gnu,            os: ubuntu-latest,  cpp: 1,           std: 1, run: 1 }
             - { target: i686-pc-windows-gnu,              os: ubuntu-latest,  cpp: 1,           std: 1, run: 1 }
+            # Disabled, see rust-lang/rust#85821
+            #- { target: asmjs-unknown-emscripten,         os: ubuntu-latest,                    std: 1, run: 1 }
+            #- { target: wasm32-unknown-emscripten,        os: ubuntu-latest,                    std: 1, run: 1 }
```

Comment:
since they are commented out, I feel like the readme should reflect the fact that it's not published.